### DISCUSSION TITLE
Recently inner page links as <h2 id="aaa"> are not working

### DISCUSSION THREAD

**Discussion by s5uishida** [community/community#68173] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hi Community,

Recently, for example, inner page links as follows are not working. (using Firefox and Chrome)
```
- [Changelog (summary)](#changelog)

...

<h2 id="changelog">Changelog (summary)</h2>
```
Is there any solution?

**Comment by seyLu** [comment, marked as answer]
#### Hi @s5uishida,

Headers are automatically converted into links in Github Markdown:
![image](https://github.com/community/community/assets/98249191/3c6cee6f-5502-4251-a5ed-095bd308a974)
![image](https://github.com/community/community/assets/98249191/e911c613-590f-4883-9236-c10f3974eaf4)

What you can do instead, if you want to customize the anchor link, is to create a hidden anchor link on top/below the elem:
```md
<a id="changelog"></a>

## Changelog (summary)
``` 

For example:
![image](https://github.com/community/community/assets/98249191/e0e4ac9c-3219-4621-b35d-a5fde0772c73)
![image](https://github.com/community/community/assets/98249191/6877ea3a-a4b1-42d7-896c-553e8c492885)
![image](https://github.com/community/community/assets/98249191/4c641082-ff9b-4a75-98f9-e6ce23689c53)

**Reply by s5uishida** [reply]
Hi @seyLu 

Thank you very much! So it will be very helpful.
